.github/workflows/stale.yml

```diff
@@ -24,15 +24,11 @@ jobs:
           This issue has been automatically marked as stale because it has not had
           recent activity. If you think this bug should stay open, please comment on 
           the issue with further details. Thank you for your contributions.
-        stale-issue-label: 'state: stale'
+        stale-issue-label: 'state:stale'
```

Comment:
perhaps a different issue/ticket/discussion, but after `state:verified` is renamed to `qa:verified` (as per spreadsheet comments) and the others removed, `state:stale` will be the only `state:xxx` label remaining... so could just be `stale`.

Reply:
Filed https://github.com/mozilla/addons/issues/14826